### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#46] [Comments]
# about/

喜欢捣鼓电脑，UseR，喜欢阅读，重度使用 Kindle。甘愿做一个平庸的人，走在成为我自己的道路上。 [&hellip;] 致虚极，守静笃。
[&hellip;] 万物并作，吾以观复。
[&hellip;] 夫物芸芸，各复归其根。
[&hellip;] 归根曰静，是谓复命。
[&hellip;] 复命曰常，知常曰明。
[&hellip;] — 《道德经》第十六章
[&hellip;] 世界上只有 &hellip;

https://shitao5.org/about/

**Comment by Paulwisley** [comment]
真好啊

**Comment by YYJ-python** [comment]
很不错，搜duckdb来到了这里；我也是做数据工作的

**Reply by Shitao5** [reply]
嘿嘿你好呀😉

**Reply by Tangweiwei227** [reply]
我也是搜duckdb来到了这里哈哈

**Reply by Shitao5** [reply]
哈哈你好你好😀

**Reply by Amo-nick** [reply]
哈哈哈，同duckdb

**Comment by QYD720** [comment]
网站做的很漂亮！

**Reply by Shitao5** [reply]
谢谢夸奖呀！[Hugo Paged](https://paged.yihui.org/about/) 主题占大半功劳哈哈。

**Comment by shenweiyan** [comment]
跟益辉大大同款的博客，真不错。

**Reply by Shitao5** [reply]
哈哈就是用了益辉大大开发的 [Hugo Paged](https://paged.yihui.org/about/) 主题。经多次折腾后确认益辉大大的产品可以[闭眼入](https://zhidao.baidu.com/question/1708840817524377300.html)，也不用再捣鼓主题啥玩意儿的，专注于内容，写就完事了。

**Comment by mbsky916** [comment]
看了你的博客首页，纯小白，学习自己搭建。目前摸索中

**Reply by Shitao5** [reply]
好耶，加油呀，有问题我们随时交流！

**Comment by Liam-Zhong** [comment]
路过 真不错

**Reply by Shitao5** [reply]
你好呀！😉

**Comment by eryajf** [comment]
偶然至此，看到了你也经常看二丫讲梵，只想说，你真的很有品味🐶

**Reply by Shitao5** [reply]
破案了：我们都是有品味的人😉

**Reply by eryajf** [reply]
再破一案，请点我的主页看我个人资料有惊喜🤣

**Reply by Shitao5** [reply]
此案莫非是早餐案，人在工位，早餐已吃👀

**Reply by eryajf** [reply]
哥哥，俺是二丫讲梵他本人

**Reply by eryajf** [reply]
感谢你的认可与喜欢

**Reply by Shitao5** [reply]
哈哈，原来是双龙戏猪案，此案不成立，俺早已识破。俺还知道龙哥正在为一个数据统计的反差而疑惑并报告 BUG 呢😎

**Reply by eryajf** [reply]
哈哈哈
扫噶
尴尬
我成透明的了
